templates/cli/index.js.twig

```diff
@@ -36,6 +36,7 @@ program
   })
   .showSuggestionAfterError()
 {% if sdk.test != "true" %}
+  .addCommand(whoami)
```

Comment:
I remember commands used to support `--json` or something like that. When used, instead of visual table, it gives JSON in response. Please check if that is the case, and if so, lets make sure this command also supports it

Reply:
![image](https://github.com/appwrite/sdk-generator/assets/316103/49717057-742d-4fbe-881d-128203960ee5)